### ISSUE TITLE
conda fails to solve using requirements.txt

### ISSUE THREAD

**Issue by menoldmt** [NICHD-BSPC/chrom-name-mappings#4]
as per \# 2 in the README.md,

running `conda create -n chrom-name-mappings --file requirements.txt --channel bioconda --channel conda-forge`

Gives:
```
LibMambaUnsatisfiableError: Encountered problems while solving:
  - package snakemake-4.4.0-py35_0 requires dropbox >=7.2.1, but none of the providers can be installed
  - nothing provides _python_rc needed by python-3.12.0rc3-rc3_hab00c5b_1_cpython

Could not solve for environment specs
The following packages are incompatible
├─ biopython is installable with the potential options
│  ├─ biopython [1.66|1.67|1.68|1.69|1.70] would require
│  │  └─ python 2.7* , which can be installed;
│  ├─ biopython [1.66|1.67|1.68|1.69|1.70] would require
│  │  └─ python 3.5* , which can be installed;
│  ├─ biopython [1.66|1.67|1.69|1.70] would require
│  │  ├─ mmtf-python with the potential options
│  │  │  ├─ mmtf-python 1.0.2 would require
│  │  │  │  └─ python 2.7* , which can be installed;
│  │  │  ├─ mmtf-python 1.0.2 would require
│  │  │  │  └─ python 3.4* , which can be installed;
│  │  │  ├─ mmtf-python 1.0.2 would require
│  │  │  │  └─ python 3.5* , which can be installed;
│  │  │  └─ mmtf-python [1.0.10|1.0.4|...|1.1.3] conflicts with any installable versions previously reported;
│  │  └─ python 3.6*  with the potential options
│  │     ├─ python [3.6.0|3.6.1|...|3.6.9] conflicts with any installable versions previously reported;
│  │     └─ python [3.6.0a3|3.6.0a4|...|3.6.0rc1], which can be installed;
│  ├─ biopython 1.68 would require
│  │  └─ python 3.4* , which can be installed;
│  └─ biopython [1.70|1.71|...|1.84] conflicts with any installable versions previously reported;
├─ python >=3.6  is not installable because there are no viable options
│  ├─ python [3.6.0|3.6.1|...|3.6.9] conflicts with any installable versions previously reported;
│  ├─ python [3.10.0|3.10.1|...|3.9.9] conflicts with any installable versions previously reported;
│  └─ python [3.12.0rc3|3.13.0rc1|3.13.0rc2] would require
│     └─ _python_rc, which does not exist (perhaps a missing channel);
└─ snakemake 4.4.0  is not installable because it requires
   └─ dropbox >=7.2.1 , which conflicts with any installable versions previously reported.
```

**Comment by esnaultc** [comment]
I updated the environment requirements in branch [v1.1](https://github.com/NICHD-BSPC/chrom-name-mappings/tree/v1.1), it works on my machine with `mamba` and `conda`.

You could switch to that new branch if you want to try before it is merged to `master`.

**Comment by esnaultc** [comment]
The updates were merged into `master` #5, closing.
